cobalt/browser/cobalt_content_browser_client.cc

```diff
@@ -48,8 +49,10 @@ class CobaltBrowserMainParts : public content::ShellBrowserMainParts {
 
   // ShellBrowserMainParts overrides.
   int PreCreateThreads() override {
-    // TODO(b/372559349): setup metrics similarly to what SetupMetrics() does
-    // when called from ChromeBrowserMainParts::PreCreateThreadsImpl().
+    metrics_ = std::make_unique<CobaltMetricsServiceClient>();
+    // TODO(b/372559349): Double check that this initializes UMA collection,
+    // similar to what ChromeBrowserMainParts::StartMetricsRecording() does.
+    metrics_->Start();
     return ShellBrowserMainParts::PreCreateThreads();
```

Comment:
Can you mention PreMainMessageLoopRun as the preferred function to call StartMetricsRecording here? I don't want to forget this when I implement it.